src/czbenchmarks/metrics/utils.py

```diff
@@ -148,6 +148,37 @@ def mean_fold_metric(results_df, metric="accuracy", classifier=None):
     return df[metric].mean()
 
 
+def classifier_metric(results_df, metric: str, **kwargs):
```

Comment:
nit: rename to indicate the function is simply extracting a metric, not computing it?  and it's general enough that maybe it's even useful outside of classifier metrics, in the future, so could be name generically.